README.md

```diff
@@ -1,3 +1,37 @@
 # SDK JS Schematics :: Demo APP ✨
 
-To start the demo application and e2e test select the desired version, or read [Angular 14 DEMO APP](/angular14-demo-app/README.md).
+## Table of Contents
+
+-   [Introduction](#introduction)
+-   [About](#about)
+-   [How it Works](#How-it-works)
+-   [Getting started](#getting-started)
+-   [License](#license)
+
+## Introduction
+
+This project serves as a demonstration of the ESMF Schematics Project in an Angular application.
+
+## About
+
+The primary goal of this project is to illustrate the generation and interaction of different tables within an Angular application using ESMF Schematics. This complete solution demonstrates not just the ability to generate tables dynamically but also outlines how to interact with them effectively.
+
+## How it Works
+The ESMF Schematics project is leveraged here to generate a variety of tables in the Angular application. You can interact with these tables, exploring features such as sorting, filtering, and pagination. This demonstration highlights the practicality of ESMF Schematics in creating powerful, interactive elements in an Angular application.
+
+## Getting Started
+
+```bash
+git clone https://github.com/eclipse-esmf/esmf-sdk-js-schematics-demo.git
+```
+
+To start the demo application and e2e test select the desired version, or read [Angular 14 DEMO App](/angular14-demo-app/README.md).
```

Comment:
What does "select the desired version" mean? What is someone supposed to run?

Reply:
Rewrite this sentence.